content/en/blog/2023/otel-end-user-discussions-february-2023.md

```diff
@@ -0,0 +1,187 @@
+[article](https://ref.otel.help/otel-collector-ops/) discussing this.
+
+#### 2 - Timeline for opamp extension
+
+**Q:** Is there any timeline for implementing the opamp spec for agent
+management?
+
+**A:** It's not a top priority as of now. It would be good to have a maintainer
+from the community for opamp. You can track the issue
+[here](https://github.com/open-telemetry/opentelemetry-collector-contrib/issues/16462).
+
+#### 3 - Buffer capabilities of OTel collector
+
+**Q:** What are some backup/retry buffer capabilities of OTel collectors when
+endpoints are unavailable?
+
+**A:** There is an experimental
+[storage extension](https://github.com/open-telemetry/opentelemetry-collector/tree/main/extension/experimental/storage)
+that is currently under development to support buffering and data persistence.
+
+### OpenTelemetry Language API & SDKs
+
+#### 1 - Timeline for Go SDK
+
+**Q:** What is the timeline for full specification compliance for OTel Go SDK?
+
+**A:** In the Go Otel SDK, the current progress is mostly around metrics. The
+logging development is frozen. Major work is being done in metrics sdk. Go
+metrics progress can be tracked
+[here](https://github.com/open-telemetry/opentelemetry-go/projects?query=is%3Aopen).
+Once metrics are done, logs will be taken care of.
+
+### OpenTelemetry Traces
+
+#### 1 - Sampling for traces
+
+**Q:** Is there a way to sample traces based on the span counts? Example:
+Drop/truncate traces which have more than 1000 spans in a single trace.
+
+**More context:** Sometimes, due to issues in the application itself, some
+traces generate a lot of spans that are not needed. Is there a way in
+OpenTelemetry to control this? More specifically, is there a way in which we can
+set up a condition where if a certain trace has more than ‘n’ number of spans,
+we can drop or truncate the number of spans.
+
+**A:** Tail-based sampling processor provides users with a bunch of sampling
+policies. Span count is one such policy. You can also combine multiple policies.
+Here's the link to tail
+[sampling processor](https://github.com/open-telemetry/opentelemetry-collector-contrib/blob/main/processor/tailsamplingprocessor/README.md).
+The span count policy is based on min span count. Some users might look for some
+kind of exclusion policy.
+
+#### 2 - Use cases of span links
+
+**Q:** What are the use cases of span links?
+
+**A:** Span links are used to have an overview when there are thousands of
+spans. It helps in navigation, and answers questions like, "Where does a trace
+start and end?"
+
+### OpenTelemetry Metrics
+
+#### 1 - Supporting other metrics format
+
+**Q:** Can Otel collector support metrics generated from other libraries like
+statsd library?
+
+**A:** The OpenTelemetry Collector contrib has a lot of receivers for different
+types of metrics that can be used. For example, if you are sending out metrics
+in Prometheus format, you can configure your Otel collector to scrape Prometheus
```

Comment:
```suggestion
in Prometheus format, you can configure your Otel Collector to scrape Prometheus
```

Capitalize "Collector" for consistency

Reply:
Otel --> OTel

Reply:
@svrnm - should we hook up `textlint` into the CI checks so that it can address issues like "OTel" capitalization?